molde/render_widgets/animated_render_widget.py

```diff
@@ -1,5 +1,8 @@
 from threading import Lock
 from time import time
+from moviepy.editor import ImageSequenceClip
```

Comment:
Can you move this import to inside the "generate_video" function?

This may reduce the import times.